saleor/graphql/account/mutations/account/account_address_create.py

```diff
@@ -67,6 +68,10 @@ def perform_mutation(  # type: ignore[override]
         cleaned_input = cls.clean_input(info=info, instance=Address(), data=input)
         with traced_atomic_transaction():
             address = cls.validate_address(cleaned_input, address_type=address_type)
+            # Check for invalid fields and return errors if necessary
```

Comment:
And just small comment, it's nothing big, but I personally do not like to write comments that are redundant. By that I mean, you named your methods correctly, and the code is already self explanatory.

Reply:
@kadewu Thanks for the feedback. I'll keep this in in mind.